### PR TITLE
Update nextjs config in SSR docs

### PR DESCRIPTION
**Pull request by reidbarber**
Codesandbox with this config if anyone wants to verify: https://codesandbox.io/s/rsp-3-16-4-with-next-js-iey61w

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/dev/docs/pages/react-spectrum/ssr.mdx [hunk 1]
```diff
@@ -103,6 +104,7 @@ const withTM = require("next-transpile-modules")([
   "@react-spectrum/well",
   "@spectrum-icons/ui",
   "@spectrum-icons/workflow"
+  "@spectrum-icons/workflow/node_modules/@react-spectrum/icon",
```
Comment (reidbarber) [inline packages/dev/docs/pages/react-spectrum/ssr.mdx:107]: Not sure what change made this need to be added, but I couldn't build without it.

Comment (snowystinger) [reply packages/dev/docs/pages/react-spectrum/ssr.mdx:107]: I think it can be removed now, https://github.com/adobe/react-spectrum/pull/2933

Comment (reidbarber) [reply packages/dev/docs/pages/react-spectrum/ssr.mdx:107]: It's still giving me issues without this on 3.16.4: 

> ./node_modules/@spectrum-icons/workflow/node_modules/@react-spectrum/icon/dist/main.css
Global CSS cannot be imported from within node_modules.
Read more: https://nextjs.org/docs/messages/css-npm
Location: node_modules/@spectrum-icons/workflow/node_modules/@react-spectrum/icon/dist/module.js

Comment (snowystinger) [reply packages/dev/docs/pages/react-spectrum/ssr.mdx:107]: can you determine why? did the workflow icon pinning update not go through with 3.16.4 and @spectrum-icons/workflow@3.2.4?

**Comment by reidbarber** [comment]
Closing in favor of #3140

**Pull request closed without merging**